azkaban-common/src/main/java/azkaban/executor/ExecutorHealthChecker.java

```diff
@@ -71,11 +72,13 @@ public ExecutorHealthChecker(final Props azkProps, final ExecutorLoader executor
 
   public void start() {
     logger.info("Starting executor health checker.");
-    this.scheduler.scheduleAtFixedRate(() -> checkExecutorHealth(), 0L, this.healthCheckIntervalMin,
+    this.scheduler.scheduleAtFixedRate(this::checkExecutorHealthQuietly, 0L,
+        this.healthCheckIntervalMin,
         TimeUnit.MINUTES);
   }
 
   public void shutdown() {
+    logger.info("Terminating executor health checker.");
```

Comment:
The function name is shutdown and message says 'Terminating'. Possible to choose one of the two words and make it consistent?

Shutdown has a meaning of a more orderly than 'terminating.'